plugin-ozone/pom.xml

```diff
@@ -91,7 +91,7 @@ limitations under the License.
         <dependency>
             <groupId>org.apache.ozone</groupId>
             <artifactId>ozone-common</artifactId>
-	        <version>${ozone.version}</version>
+	        <version>1.4.0-SNAPSHOT</version>
```

Comment:
I had to hardcode the version in order to test it locally as some changes are also done on the Ozone side. How are we going to handle it?